### PR TITLE
Support new Inspector select api and a few other minor tweaks

### PR DESCRIPTION
**Pull request by jacob314**
to prepare for main HotUI CL.
Fixes https://github.com/flutter/flutter-intellij/issues/4052

### CODE REVIEW

**Comment by jacob314** [comment]
Fixes https://github.com/flutter/flutter-intellij/issues/4052

**Comment by jacob314** [comment]
The similar CL to the DevTools landed a few weeks back.

File: src/io/flutter/view/FlutterView.java [hunk 1]
```diff
@@ -321,8 +340,10 @@ public void debugActive(@NotNull FlutterViewMessages.FlutterDebugEvent event) {
     }
     else {
       whenCompleteUiThread(
-        InspectorService.create(app, app.getFlutterDebugProcess(), app.getVmService()),
+        app.getFlutterDebugProcess().getInspectorService(),
         (InspectorService inspectorService, Throwable throwable) -> {
+          // XXX remove
+          // app.getFlutterDebugProcess().setInspectorService(inspectorService);
```
Comment (kenzieschmoll) [inline src/io/flutter/view/FlutterView.java]: nit: remove comment?

File: src/io/flutter/vmService/ServiceExtensions.java [hunk 2]
```diff
@@ -132,6 +164,8 @@
     slowAnimations,
     togglePlatformMode,
     toggleSelectWidgetMode,
+    toggleOnDeviceWidgetInspector,
+    enableOnDeviceInspector,
```
Comment (kenzieschmoll) [inline src/io/flutter/vmService/ServiceExtensions.java]: nit: looks like this list is alphabetized. Put enableOnDeviceInspector in correct position

File: src/io/flutter/view/FlutterView.java (hunk 1 above)
Comment (jacob314) [reply src/io/flutter/view/FlutterView.java]: done

File: src/io/flutter/vmService/ServiceExtensions.java (hunk 2 above)
Comment (jacob314) [reply src/io/flutter/vmService/ServiceExtensions.java]: done

**Pull request merged**